src/window.rs

```diff
@@ -50,7 +50,9 @@ mod imp {
                 "sidebar.begin-chats-search",
                 None,
             );
-
+            klass.install_action("win.paste", None, move |widget, _, _| {
```

Comment:
This overwrites the default shortcut to paste, so basically paste is broken with everything but the image clipboard.